task/rhacs-deployment-check/3.71/rhacs-deployment-check.yaml

```diff
@@ -64,20 +74,26 @@ spec:
           value: $(params.insecure-skip-tls-verify)
         - name: OUTPUT
           value: $(params.output_format)
-        - name: ROX_API_TOKEN
-          valueFrom:
-            secretKeyRef:
-              name: $(params.rox_api_token)
-              key: rox_api_token
+        #- name: ROX_API_TOKEN
+        #  valueFrom:
+        #    secretKeyRef:
+        #      name: $(params.rox_api_token)
+        #      key: rox_api_token
+        - name: ROXCTL_CONFIG_DIR
+          value: /roxctl-config/$(params.roxctl-config-subdir)
+        - name: ROX_TOKEN_FILE
+          value: $(params.token_file)
         - name: ROX_CENTRAL_ENDPOINT
-          valueFrom:
-            secretKeyRef:
-              name: $(params.rox_central_endpoint)
-              key: rox_central_endpoint
+          value: $(params.rox_central_endpoint)
+          #valueFrom:
+          #  secretKeyRef:
+          #    name: $(params.rox_central_endpoint)
+          #    key: rox_central_endpoint
       args:
         - deployment
         - check
         - --endpoint=$(ROX_CENTRAL_ENDPOINT)
```

Comment:
let's move this also to use `ROX_ENDPOINT` as environment variable then we can skip setting the flag.